frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -90,12 +90,29 @@ export const menuActionStart = (
 ): KebabOption => {
   const title = 'Start Virtual Machine';
   return {
-    hidden:
-      vmStatusBundle?.status?.isImporting() ||
-      vmStatusBundle?.status?.isMigrating() ||
-      isVMRunningOrExpectedRunning(vm),
+    hidden: vmStatusBundle?.status?.isMigrating() || isVMRunningOrExpectedRunning(vm),
     label: title,
-    callback: () => startVM(vm),
+    callback: () => {
+      if (!vmStatusBundle?.status?.isImporting()) {
+        startVM(vm);
+      } else {
+        confirmModal({
+          title,
+          message: (
+            <>
+              <p>
+                This virtual machine will start as soon as the import has been completed. If you
+                proceed you will not be able to change this option.
+              </p>
+              Are you sure you want to start <strong>{getName(vm)}</strong> in namespace{' '}
+              <strong>{getNamespace(vm)}</strong> after it has imported?
+            </>
+          ),
+          btnText: _.capitalize(VMActionType.Start),
+          executeFn: () => startVM(vm),
+        });
+      }
+    },
     accessReview: asAccessReview(kindObj, vm, 'patch'),
   };
 };
```

Comment:
are we still okay with keeping the stop isDisabled in this release?

anyway, can you please at least use isImporting status instead of isPending there?